motionplan/kinematic_test.go

```diff
@@ -88,8 +88,11 @@ func TestForwardKinematics(t *testing.T) {
 	// Test out of bounds. Note that ComputePosition will return nil on OOB.
 	newPos = []float64{-45, 0, 0, 0, 0, 999}
 	pos, err = ComputePosition(m, &pb.JointPositions{Values: newPos})
-	test.That(t, pos, test.ShouldBeNil)
-	test.That(t, err, test.ShouldNotBeNil)
+	checkOrien := &spatial.R4AA{Theta: -2.48798057005674, RX: 0.23071941493324336, RY: -0.7100813450467474, RZ: 0.6652465971273088}
```

Comment:
[nit] Seems like this could more closely match the form of what is taking place on L79-86.